ruby/ql/test/library-tests/ast/ValueText.expected

```diff
@@ -355,29 +357,27 @@
 | gems/test.gemspec:9:3:9:12 | __synth__0 | https://github.com/github/codeql-ruby |
 | gems/test.gemspec:9:19:9:57 | "https://github.com/github/cod..." | https://github.com/github/codeql-ruby |
 | literals/literals.rb:2:1:2:3 | nil | nil |
-| literals/literals.rb:3:1:3:3 | NIL | NIL |
+| literals/literals.rb:3:1:3:3 | NIL | nil |
 | literals/literals.rb:4:1:4:5 | false | false |
-| literals/literals.rb:5:1:5:5 | FALSE | FALSE |
+| literals/literals.rb:5:1:5:5 | FALSE | false |
 | literals/literals.rb:6:1:6:4 | true | true |
-| literals/literals.rb:7:1:7:4 | TRUE | TRUE |
+| literals/literals.rb:7:1:7:4 | TRUE | true |
 | literals/literals.rb:10:1:10:4 | 1234 | 1234 |
-| literals/literals.rb:11:1:11:5 | 5_678 | 5_678 |
+| literals/literals.rb:11:1:11:5 | 5_678 | 5678 |
 | literals/literals.rb:12:1:12:1 | 0 | 0 |
-| literals/literals.rb:13:1:13:5 | 0d900 | 0d900 |
-| literals/literals.rb:16:1:16:6 | 0x1234 | 0x1234 |
-| literals/literals.rb:17:1:17:10 | 0xdeadbeef | 0xdeadbeef |
-| literals/literals.rb:18:1:18:11 | 0xF00D_face | 0xF00D_face |
```

Comment:
We appear to have lost these two hexadecimal literals.

Reply:
Good spot, I'll fix.